### PR TITLE
add initial api setup with hello world endpoint

### PR DESCRIPTION
**Pull request by backevik**

### CODE REVIEW

File: api/config/.env.local [hunk 1]
```diff
@@ -0,0 +1 @@
+DBHost=http://localhost:5432
```
Comment (emilieanthony) [inline api/config/.env.local:1]: Should it be a new line here since it is the end of a file?

File: api/internal/svc/carbon_dioxide/service.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package carbon_dioxide
```
Comment (emilieanthony) [inline api/internal/svc/carbon_dioxide/service.go:1]: Nice naming!

File: api/config/.env.local (hunk 1 above)
Comment (backevik) [reply api/config/.env.local:1]: Yes! Good catch! 👌 Will update later this evening.

Comment (backevik) [reply api/config/.env.local:1]: I realized that this file should even be removed as it's not used anymore.

Comment (emilieanthony) [reply api/config/.env.local]: Great!

**Review by emilieanthony — APPROVED** [review]
Impressed, I approve!

**Pull request merged**